File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        HashMap<Character, Integer> map = new HashMap<>();

        for(int i = 0; i < sentence.length(); i++){
            if(!map.containsKey(sentence.charAt(i)))
                map.put(sentence.charAt(i), 1);
            else
                map.put(sentence.charAt(i), map.get(sentence.charAt(i)) + 1);
        }
        return (map.size() == 26);
    }
}

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.swift
class Solution {
    func checkIfPangram(_ sentence: String) -> Bool {
        var dictionary = [Character : Int]()
        
        for i in sentence {
            if dictionary[i] == nil {
                dictionary[i] = 1
            } else {
                dictionary[i]! += 1
            }
        }
        
        return dictionary.count == 26 ? true : false
    }
}